plugins/inputs/win_perf_counters/README.md

```diff
@@ -181,6 +212,13 @@ This must be specified for every counter you want the results of, or use
 `["*"]` for all the counters of the object, if the `UseWildcardsExpansion` param
 is set to `true`.
 
+#### Sources (Object)
+
+(Optional)
+
+Overrides the [Sources](#sources) global parameter for current performance object.
+See [Sources](#sources) description for more details.
+
```

Comment:
```suggestion
Example: 
 ` [[inputs.win_perf_counters.object]]
    ObjectName = "Memory"
    Counters = ["Available Bytes"]
    Instances = ["------"]
    Measurement = "win_mem"
    Sources = ["localhost"]`
```

@sspaink Having an example on how to override is helpful in my opinion, let me know your thoughts.

Reply:
We can add this later I think.